implementation/src/main/java/io/smallrye/jwt/auth/jaxrs/JWTAuthenticationFilter.java

```diff
@@ -62,6 +61,7 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
 
             if (bearerToken != null) {
                 try {
+                    requestContext.setProperty(HAS_JWT, true);
```

Comment:
I've looked at this filter, it catches the exception, logs, and continues, it needs to return 401 immediately, so a new property, and a new optional filter won't be needed

Reply:
The idea was to follow the same structure we had with the others filters. Sure, I can change it.

Reply:
Yeah, if we have a token and it failed to parse then relying on some follow up filters to block such an invalid token will not be very safe...

Reply:
@radcortez In quarkus we may need to relax things a bit, here we have a proactive authentication, but in Quarkus Stuart added an option to authenticate lazily, but we can discuss it later in Quarkus

Reply:
Sure. Quarkus doesn't use this filter anyway.